Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/Microsoft.Toolkit.Forms.UI.Controls.csproj

```diff
@@ -2,33 +2,24 @@
 
   <PropertyGroup>
     <TargetFramework>net462</TargetFramework>
-    <RootNamespace>Microsoft.Toolkit.Win32.UI.Controls</RootNamespace>
-    <AssemblyName>Microsoft.Toolkit.Win32.UI.Controls</AssemblyName>
+    <RootNamespace>Microsoft.Toolkit.Forms.UI.Controls</RootNamespace>
```

Comment:
Removing the TargetPlatformMinVersion would be fine, except the XamlHost controls require 17692 of SDK (RS5 build), while the WebView requires 17134, an RS4 build. XAML Islands controls and Web View cannot live together as each have different SDK / OS requirements.

Reply:
Sounds like we need to move WebView & WebViewCompatible to their own project?

Reply:
Yes, I think so unless the XAML team is willing to backport their changes to 17134